index.html

```diff
@@ -3514,6 +3514,58 @@ <h3>Reserved Extension Points</h3>
         </p>
 
       </section>
+
+      <section class="normative">
+        <h3>Ecosystem Compatibility</h3>
+
+        <p>
+There are a number of digital credential formats that do not natively use the
+data model provided in this document, but are aligned with a number of concepts
+in this specification. At the time of publication, examples of these digital
+credential formats include pure JSON Web Tokens (JWTs), CBOR Web Tokens (CWTs),
+ISO-18013-5 (mDLs), and Authentic Chained Data Containers (ACDCs).
```

Comment:
This section is normative, these should be references to specification of equal standing, or they should be omitted.

Reply:
You can make informative references to external specifications in a normative section. 

There are zero normative requirements around these specifications and thus the references to them are informative in nature. They are merely provided as examples.

Reply:
But those specification do have bearing here, because they represent the "input media types".

Reply:
Don't see value in listing types unless we are also listing registered media types

Reply:
I support the inclusion of these examples.  As they are informative references they aren't prohibited (as mentioned above) and their inclusion represents an acknowledgement that members of those communities joined the W3C and this working group based on assertions that this specification would represent a "big tent".  In the absence of any other language or work items being adopted in support of a big tent, I suggest these examples remain in the PR.

Reply:
VCDM -> data model (JSON-LD/JSON)
JWT -> Signature (JWS compact serialised)
CBOR -> data format

For mdl and ACDC I don't know.

We should compare same things.
VCDM (DM == Data Model) is defined in this document.

Base data format is set as JSON-LD (which is also a JSON). This can be compared with COSE/COSE-LD?. This transformation is: same data model in different formats.

JWT (a profile of JWT) should be compared to JAdES, Data Integrity Proofs and other securing mechanisms.

Reply:
@ChristopherA might like to see Gordian Envelope included here, they were part of the original Face2Face conversation and subsequent WG calls.

Reply:
> @ChristopherA might like to see Gordian Envelope included here, they were part of the original Face2Face conversation and subsequent WG calls.

Yes, agreed, we could use this reference at IETF: https://datatracker.ietf.org/doc/draft-mcnally-envelope/

Reply:
I am supportive of including examples, but they need to have references, or I will object... If the example is not mature enough to reference, it's not mature enough to comment on in our spec.

It's also not great to mix examples of varying quality if they are vastly different in maturity, because it anchors that they are the "same level of maturity" in the mind of the reader.

This is not about tents, it's about claiming community drafts and ISO standards are the same thing, or equally noteworthy... that should not even be hinted.

Reply:
Per the 2023-07-18 special topic call, came to consensus on adding links to the referenced specifications, I have added links to the specifications in this PR (and clarified some of the language, as requested): 088020d9028481a168fa34b48f9cc4df1c693a81